lib/schema.js

```diff
@@ -661,15 +666,22 @@ Schema.prototype.execPostHandlers = function(event, mdlInst, callback) {
   try {
     var postHandlers = this.postHandlers[event];
     for (var i = 0; i < postHandlers.length; ++i) {
-      postHandlers[i](mdlInst);
+      // After an action has taken place we don't really
+      // need this callback, but we will add it for consistency
+      // in the API: post handlers should expect to always get
+      // 2 arguments, mdlInst and callback.
+      var doNext = function () { };
+
+      postHandlers[i].call(mdlInst, mdlInst, doNext);
     }
+
     callback(null);
   } catch (e) {
     callback(e);
   }
 };
 
-var supportedEvents = ['validate', 'save', 'remove'];
+var supportedEvents = ['validate', 'save', 'load', 'remove'];
```

Comment:
Note this change -- docs in model instance said that "load" was supported, but it wasn't, because "load" wasn't in this array.